spec/models/reports/monthly_bulk_serializer_spec.rb

```diff
@@ -11,14 +11,12 @@ module Reports
     describe "#to_csv" do
       it "return a string in CSV format with complete exemptions details for the given month" do
         create_list(:registration, 3)
-
-        # TODO: Solve this at a factory level and deal with the cascade of fixes
-        WasteExemptionsEngine::RegistrationExemption.update_all(registered_on: Date.today)
-
+        # The registration factories default to creating 3 exemptions per-registration factory.
+        total_exemptions = 3 * 3
```

Comment:
I'm not sure about doing this over `WasteExemptionsEngine::RegistrationExemption.count` - it feels like we are making the test more fragile as if the factory changes how many exemptions it has, then it'll break.

Reply:
Yeah. I think if we change line 13 (https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/252/files#diff-f71d8ac0837fc977208704135536563dR13) to clearly add a number of registration exemptions instead that would make this clearer. WDYT?

Reply:
@cintamani Sounds good to me :)